scripts/setup_ubuntu.sh

```diff
@@ -91,12 +106,15 @@ else
 fi
 
 run_as_original_user pip3 install --upgrade "pip$PIP_VERSION" "setuptools$SETUPTOOLS_VERSION"
-run_as_original_user pip3 install conan==1.36 PyGithub packaging
+run_as_original_user pip3 install testresources conan==1.36 PyGithub packaging
+# since we install as $USER, pip installs ~/.local/bin,
+# but this is not in the default ubuntu $PATH. Add it.
+run_as_original_user export PATH=$PATH:$HOME/.local/bin
```

Comment:
I think you are trying to add it to the user path. But this will not actually do anything (because of how shells work). Also I'm pretty sure `.local/bin` is in the user path by default at least if the directory exists.

Reply:
bah, you are definitely right about export. 
re: $PATH, when I ran these scripts on my 21.04 box, the next conan script failed because it could not find `conan` despite it being available in `~/.local/bin`. 
I suspect if I had opened a fresh shell, whatever script adds `~/.local/bin` to $PATH automagically would have done its job. 
anyway, I'll drop this change for now